OpenTissue/gpu/image/io/image_read.h

```diff
@@ -1,91 +1,180 @@
-#ifndef OPENTISSUE_GPU_IMAGE_IO_IMAGE_READ_H
-#define OPENTISSUE_GPU_IMAGE_IO_IMAGE_READ_H
 //
 // OpenTissue Template Library
 // - A generic toolbox for physics-based modeling and simulation.
 // Copyright (C) 2008 Department of Computer Science, University of Copenhagen.
 //
 // OTTL is licensed under zlib: http://opensource.org/licenses/zlib-license.php
 //
-#include <OpenTissue/configuration.h>
+#pragma once
```

Comment:
Does this really work on Mac/Linux?

As I recall this is non-standard?

The purpose of the ifndef and define blocks are to achieve the same effect but using standard only

Reply:
Yes, it is supported by the compilers that mater (gcc, clang, msvc), and yes is non-standard. It has become second nature for me to use the pragma. Will revert.